code/__HELPERS/filters.dm

```diff
@@ -141,7 +141,7 @@ GLOBAL_LIST_INIT(master_filter_info, list(
 
 //Helpers to generate lists for filter helpers
 //This is the only practical way of writing these that actually produces sane lists
-/proc/alpha_mask_filter(x, y, icon/icon, render_source, flags)
+/proc/alpha_mask_filter(x, y, icon/icon, atom/render_source, flags)
```

Comment:
Shouldn't be changed, ``/image`` is also valid but ``/datum`` is not so we don't have a shared parent to type this off, and you use it yourself with mutables later in cracked.dm